src/main/java/org/opentripplanner/client/OtpApiClient.java

```diff
@@ -63,6 +63,7 @@ public TripPlan plan(TripPlanParameters req) throws IOException {
             req.time().toLocalDate().toString(),
             req.time().toLocalTime().truncatedTo(ChronoUnit.SECONDS).toString(),
             req.searchDirection().isArriveBy(),
+            req.searchWindow().map(sw -> "searchWindow : %d".formatted(sw.toMinutes())).orElseGet(""),
```

Comment:
I just realised that you have to pass in the search window as seconds, not minutes, so use `toSeconds` instead.

https://docs.opentripplanner.org/api/dev-2.x/graphql-gtfs/queries/plan

Reply:
I was looking via different GraphiQL instance and there were minutes in the documentation. Don't know why the different but will change it.